ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -678,6 +693,20 @@ pub(crate) struct FuncType {
     is_vararg: bool,
 }
 
+impl FuncType {
+    pub fn arg_ty_idxs(&self) -> &Vec<TypeIdx> {
```

Comment:
`pub(crate)` unless we *really* have to `pub` these (which I don't think we do).

Reply:
`pub(crate)` didn't work. Let me try again.

Reply:
I could have sworn that I was getting an error at some point, but maybe that was unrelated. Seems to be working now. Fixed in [c8ff9d9](https://github.com/ykjit/yk/pull/988/commits/c8ff9d93537605107934da4253109b6ce76beb20)